client/blocks/eligibility-warnings/index.tsx

```diff
@@ -117,13 +121,34 @@ export const EligibilityWarnings = ( {
 	const showThisSiteIsEligibleMessage =
 		isEligible && 0 === listHolds.length && 0 === warnings.length;
 
+	const blockingMessages = getBlockingMessages( translate );
+
 	return (
 		<div className={ classes }>
 			<QueryEligibility siteId={ siteId } />
 			<TrackComponentView
 				eventName="calypso_automated_transfer_eligibility_show_warnings"
 				eventProperties={ { context } }
 			/>
+			{ ! isPlaceholder && context === 'plugin-details' && hasBlockingHold( listHolds ) && (
```

Comment:
This seems to be the best strategy for this PR. That said, I feel it would be better to rethink these `HardBlockingNotice` messages. They feel weird in this modal. We should probably show only them if they're present.

Something like this, but with a better design. Any thoughts @vinimotaa?

![image](https://user-images.githubusercontent.com/1044309/192781996-848bdd19-99b9-4085-b248-e48f9fdc6667.png)

Reply:
> Which looks kind of broken.

Agree @lsl. Talking with @vinimotaa, we can go ahead now, and when possible, we'll review these errors and probably extract them from the modal.

Reply:
https://github.com/Automattic/wp-calypso/issues/68250 should be moving these into the warnings area like this:

![Screenshot(30)](https://user-images.githubusercontent.com/811776/192938440-aed31503-94a0-4f9f-8786-730a3b21e34c.png)